README.md

```diff
@@ -72,43 +74,52 @@ and grant it the same access.
     ./scripts/activate.bat   # Windows CMD only
     ```
 
-1.  Clone the repository.
+3.  Clone the repository.
 
     ```
     git clone https://github.com/Azure-Samples/app-service-msi-keyvault-python.git
```

Comment:
We are going to want to rename this repo to remove MSI from the path.  app-service-azmanagedid-keyvault-python.